wallet/CHANGES

```diff
@@ -1,4 +1,11 @@
 Dash Wallet
+v9.0.5
+* Update to dashj 19.0.5
+* Update translations for the new Send flow
+* Fix stuck currency code on the Receive screen
+* Fix overflowing labels on the Home and Advanced Security screens
+* Fix leaking balance info on the Quick Receive screen
+
```

Comment:
I think we should add these items for our records -- these aren't necessary for translated release notes.

* Update transaction error handling based on removal of BIP61 in Dash Core 19
* Fix explore DB file name for prod flavor (was same as testnet for 9.0.4)